flow-report/src/header.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {FunctionComponent} from 'preact';
+
+import {Util} from '../../report/renderer/util';
+import {FlowStepIcon, FlowStepThumbnail} from './common';
+import {getModeDescription, useFlowResult} from './util';
+
+const SIDE_THUMBNAIL_HEIGHT = 74;
+const MAIN_THUMBNAIL_HEIGHT = 112;
```

Comment:
Sorta feels too small tbh. But not gonna be an issue when we get the final screenshot in its prominent place. #13123

Reply:
Made em a little bigger